File: Confitec-Back/Confitec.Domain/Models/User/Request/InsertUsuarioRequest.cs
using Confitec.Domain.Models.Enums;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Confitec.Domain.Models.User.Request
{
    public class InsertUsuarioRequest
    {
        public string Nome { get; set; }
        public string Sobrenome { get; set; }
        public string Email { get; set; }
        public DateTime DataNascimento { get; set; }
        public EEscolaridade Escolaridade { get; set; }
    }
}

File: Confitec-Front/src/app/pages/usuarios/models/request/InsertUsuario.request.ts
import { EEscolaridade } from 'src/app/enums/EEscolaridade';

export class InsertUsuarioRequest {
  public nome: string | undefined;
  public sobrenome: string | undefined;
  public email: string | undefined;
  public dataNascimento: string | undefined;
  public escolaridade: EEscolaridade | undefined;
}
